src/gossip/service.zig

```diff
@@ -56,7 +56,7 @@ const globalRegistry = @import("../prometheus/registry.zig").globalRegistry;
 const GetMetricError = @import("../prometheus/registry.zig").GetMetricError;
 const Counter = @import("../prometheus/counter.zig").Counter;
 
-const PacketBatch = ArrayList(Packet);
+const PacketBatch = std.ArrayListUnmanaged(Packet);
```

Comment:
It is my opinion that this actually reduces the mental overhead when you consider the fact that the allocator used to alloc/free the packet batches going through a channel can now be centralized in one specific place, instead of needing to track down the origin of each packet batch through the multithreaded access of a given channel. This also has the effect that the lifetime of any allocator used for this purpose is now more obvious, which will allow us to avoid accidental UAF of allocator interfaces.